arm/Microsoft.Network/networkWatchers/.parameters/min.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```

Comment:
Location is also optional, should we remove this too?

Reply:
Yes, and no. The thing is that the default deployment is west europe for our pipelines, so vNets and other resources are deployed there. There can only be one network watch in a region pr sub, and this is currently being done by the deployment using the full parameter file. So even though it is not needed, I added it to be able to deploy it to a separate region to avoid deployment conflicts (resulting in errors and failing pipelines).

Any suggestion around this?

Reply:
Makes sense to me. I think it's good then